### DISCUSSION TITLE
Create CDS release notes details?

### DISCUSSION THREAD

**Discussion by ecki** [microsoft/openjdk#279] [Q&A] [answered]
Your release notes contain this note:

> Binaries do not have Default CDS ([JEP 341](https://openjdk.java.net/jeps/341)) archives. Run java -Xshare:dump to produce them after installation.

can you clarify a bit:

- will this be changed in the future by MS or are you waiting for upstream. Will it require installers?
- Can JLink create custom images with the archive created or included
- When using modules, is the standard class list adapted (or does it contain java.base only anyway?)
- How arch/os/cpufeature specific are those dumps, can they be created and shipped?
- The doc says you should create the dumps with same xmx size as runtime, how strict/helpful is this requirement?
- Is the shared archive mapping working/conflicting with multiple installations of the vm on the same host, if so will symlinks to w shared location help or is java_home needed?

**Comment by karianna** [comment, marked as answer]
#391

**Comment by ecki** [comment]
Thanks for submitting that ticket, will follow it there.
